core/sql/cli/Globals.h

```diff
@@ -510,6 +523,9 @@ inline
   LmLanguageManagerC *langManC_;
   LmLanguageManagerJava *langManJava_;
   NABoolean espProcess_;
+  HBaseClient_JNI *hbaseClientJNI_;
+  HBaseClient_JNI *bigtableClientJNI_;
+  MonarchClient_JNI *monarchClientJNI_;
```

Comment:
Not clear to me what bigtableClientJNI_ and monarchClientJNI_ are intended for.

Reply:
I will remove it in my next PR unless there is a need for another commit